Sources/Purchasing/OfferingsManager.swift

```diff
@@ -76,7 +111,7 @@ class OfferingsManager {
         appUserID: String,
         isAppBackgrounded: Bool,
         fetchPolicy: FetchPolicy = .default,
-        completion: (@MainActor @Sendable (Result<Offerings, Error>) -> Void)?
+        completion: (@MainActor @Sendable (Result<OfferingsResultData, Error>) -> Void)?
```

Comment:
This was also a bit tricky... I had to bubble up the requested product ids + not found product ids and this was the simplest... We could also just not add these to the event and just keep them in the store specific event (`apple_products_request` in this case). That might simplify this a bit, but might require more analysis in the data side to associate the events... I ended up just adding this for now. Lmk what you think!

Reply:
I get it, a new struct to wrap the three, but if it's useful to have this information in the event, I'd say we go with it 👍